docs/USING_GRID_REDUCERS.md

```diff
@@ -109,15 +109,15 @@ If you want to hide grid reducers from the root of your state:
 
 ````js
 import { combineReducers } from 'redux';
-import { GridRootReducer } from 'react-redux-grid';
+import { rootReducer } from 'react-redux-grid';
```

Comment:
Hmm, we probably don't want `rootReducer` here, we just want the `Reducers` right?